avro2py/codegen.py

```diff
@@ -168,7 +169,7 @@ def record_to_class(record: Record) -> ResolvedClassResult:
     class_body.append(
         ast.Assign(  # _original_schema = "..."
             targets=[ast.Name(id='_original_schema')],
```

Comment:
Likewise, here, I would like to see this be
```suggestion
            targets=[ast.Name(id='_schema')],
```
as it is no longer the "original" from the AVSC, but our resolved/modified version.